protocol/wasmbinding/bindings/msg.go

```diff
@@ -1,8 +1,11 @@
 package bindings
```

Comment:
Since this is in Golang, any reason we need to define these again, instead of just importing them from existing types? E.g `subaccounttypes.SubaccountId`